Samples/iOS-Swift/iOS-Swift.xcodeproj/project.pbxproj

```diff
@@ -665,6 +665,45 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		844A35EC2834480300C6D1DF /* Swiftlint */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = Swiftlint;
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "bash ../../scripts/run-swiftlint.sh\n";
+		};
+		844A35F328344A2700C6D1DF /* Swiftlint */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = Swiftlint;
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "bash ../../scripts/run-swiftlint.sh\n";
```

Comment:
This seems not to work for me though. When I now build this sample I get plenty of errors about Force Unwrapping Violation, which we have disabled for tests.
<img width="124" alt="Screen Shot 2022-05-18 at 11 08 46" src="https://user-images.githubusercontent.com/2443292/169002678-403b777e-046b-4a55-91e1-45edc6ed12e8.png">

Reply:
Reverted the build phase additions